src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -53,12 +54,17 @@ public static void checkUser(Request req) {
         try {
             DecodedJWT jwt = verifier.verify(token);
             Auth0UserProfile profile = new Auth0UserProfile(jwt);
+            if (!isValidUser(profile)) {
```

Comment:
This change is breaking the OTP user signup process. Do we need this check for something in particular? After a new OTP User has signed up, an Auth0 profile is available, however no `OtpUser` (or `ApiUser` or `AdminUser`) exist for that user. (The `OtpUser` is created when the user finishes the account creation wizard.)

Reply:
Hmm, that's a case that I think we would want to handle in a special way. I extracted the `isValidUser` check from `isAuthorized` because in nearly all cases, we want to stop a request in `checkUser` if it's not being made by a user.